### DISCUSSION TITLE
FIX: Special Characters coming out incorrectly in Windows

### DISCUSSION THREAD

**Discussion by NSPatton** [nss-day-cohort-47/C47-Discussions#49] [Look what I learned!]
I'm on windows 10 and none of my special characters came out properly. I found out I accidentally hit alt shift and it changed my keyboard language from English to Spanish and my / came out wrong, and my = turned into an upside down question mark. If you come across this issue make sure to hit alt shift and switch your keyboard back to English on the right of your screen and you should be golden!

**Comment by BryanNilsen** [comment]
Yeah, that was a weird ride and a fun problem to troubleshoot with you, Nick. Thanks for sharing this!!!
